concent_api/core/subtask_helpers.py

```diff
@@ -185,3 +186,22 @@ def are_subtask_results_accepted_messages_signed_by_the_same_requestor(subtask_r
         ) for subtask_results_accepted in subtask_results_accepted_list
     )
     return are_all_signed_by_requestor
+
+
+def get_one_or_none_subtasks_from_database(
+    subtask_id: str,
+    optional_condition_dict: Dict[str, str] = None,
+) -> Optional[Subtask]:
+
```

Comment:
I think this is a little bit to complicated to do a simple thing. Why not using `try/except` pattern here ? Btw. `conditions_dict` and its update is fine, I mean the `query` part.

Reply:
done